src/Tinker.jl

```diff
@@ -161,52 +182,20 @@ end
 # Moves polygon to a given location
 function move_polygon_to(p::AbstractArray, pt::XY)
     # find diff b/t start & pt; add diff to all in p
-    diff = XY(pt.x-p[1].x,pt.y-p[1].y)
-    map(n -> XY(n.x+diff.x,n.y+diff.y),p)
+    if ispolygon(p)
+        diff = XY(pt.x-p[1].x,pt.y-p[1].y)
```

Comment:
Seems like we might want to define arithmetic methods for `XY`? Something like this:
```julia
Base.:+(a::XY, b::XY) = XY(a.x + b.x, a.y + b.y)
Base.:-(a::XY, b::XY) = XY(a.x - b.x, a.y - b.y)
```
and then everywhere you can just write `diff = pt - p[1]`.

These little convenience functions are part of what make Julia fun :smile:. Would be best to make that a pull request to GtkReactive, since that's where `XY` is defined.